dwa_local_planner/src/dwa_planner.cpp

```diff
@@ -269,9 +281,8 @@ namespace dwa_local_planner {
     std::vector<geometry_msgs::PoseStamped> front_global_plan = global_plan_;
     double angle_to_goal = atan2(goal_pose.pose.position.y - pos[1], goal_pose.pose.position.x - pos[0]);
     front_global_plan.back().pose.position.x = front_global_plan.back().pose.position.x +
-      forward_point_distance_ * cos(angle_to_goal);
-    front_global_plan.back().pose.position.y = front_global_plan.back().pose.position.y + forward_point_distance_ *
-      sin(angle_to_goal);
+        std::abs(forward_point_distance_) * cos(angle_to_goal);
+    front_global_plan.back().pose.position.y = front_global_plan.back().pose.position.y + std::abs(forward_point_distance_) * sin(angle_to_goal);
```

Comment:
slightly formatting inconsistency. Maybe we should start using this for external ROS repos: https://github.com/davetcoleman/roscpp_code_format